projectgenerator/libqgsprojectgen/dataobjects/layers.py

```diff
@@ -34,6 +34,7 @@ def __init__(self, provider, uri, name, geometry_column=None, wkb_type=QgsWkbTyp
         self.__layer = None
         self.fields = list()
         self.is_domain = is_domain
+        self.is_nmrel = is_nmrel
```

Comment:
If this is true, this is a junction table? I.e. the generated table with the two foreign keys to the tables that have the N:M relation?

Reply:
Yes. According to `t_ili2db_table_prop.settings` are `ASSOCIATION`.

Reply:
Could you add a comment on this variable?

```
self.is_nmrel = is_nmrel
""""If a layer has is_nmrel set to true it is a junction table in a N:M relation. Or in ili2db terms, the table (or two foreign keys on this table?) is marked as ASSOCIATION in t_ili2db_table_prop.settings."""
```
(Please review the docstring, I'm not sure it's correct!!)